handlers/helloworld/helloWorld.go

```diff
@@ -1,23 +1,25 @@
-package handlers
+package helloworld
```

Comment:
Do we need this route anymore?

Reply:
It's not doing any harm and it's useful to just check the lambda is working